lib/minimax_rb.rb

```diff
@@ -80,4 +84,8 @@ def self.win? board, marker, opponent
   def self.board_full? board, empty_marker
     not board.include?(empty_marker)
   end
+
+  def self.all_args_included? args
```

Comment:
It's good that you thought through, documented, and tested against the possibility that someone would call into `::best_move` with incorrectly-named, or missing arguments.  If going through that process seems laborious, it can be a good indicator that there might be a way to avoid that possible misuse all together.

If the goal is to prevent callers from getting numerous parameters mixed up and passing them in the wrong order, would (required) [keyword arguments](https://thoughtbot.com/blog/ruby-2-keyword-arguments#required-keyword-arguments) provide the functionality you're looking for?  If so, Ruby itself can take care of checking arguments and giving you error messages.